### PR TITLE
[WIP] Accelerate WordPress with Rackspace CDN

### PR DESCRIPTION
**Pull request by RobinJoe**
* Rewriting the CDN procedure to remove Cloud Files references
* Replacing removed content with instructions on setting up and
  using Rackspace CDN at mcdn.rackspace. com

Addresses issue /rackspace-how-to/issues/#1579

### CODE REVIEW

**Comment by RobinJoe** [comment]
This is a Work In Progress draft. Proofreading is welcome, however I still need to test the procedure properly, and check in with James L. for SME guidance on the reliability of the procedure.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-ebc3d74cc9/how-to/)

File: content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md [hunk 1]
```diff
@@ -11,152 +11,134 @@ product: Cloud Files
 product_url: cloud-files
 ---
 
-You can enhance your WordPress blog by using Cloud Files with content delivery network (CDN) 
-technology to display content to users faster and more efficiently. You can distribute 
-common files or content such as CSS, JavaScript, images, videos and much more through a CDN, 
+You can enhance your WordPress blog by Rackspace content delivery network (CDN)
```
Comment (asettle) [inline content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md]: with Rackspace content delivery *

File: content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md [hunk 2]
```diff
@@ -11,152 +11,134 @@ product: Cloud Files
...
+technology to display content to users faster and more efficiently. You can distribute
+common files or content such as CSS, JavaScript, images, videos and much more through a CDN,
 which serves the content from the closest edge server to the end user.
 
-The following steps are best practices and have been tested and verified to optimize 
-performance on a default installation of WordPress with the default theme. This process was 
-last tested using WordPress version 3.8 and W3 Total Cache plugin version 0.9.3.
+The following steps are best practices and have been tested and verified to optimize
+performance on a default installation of WordPress with the default theme.
+This process was last tested using WordPress version 3.8 and W3 Total Cache plugin
```
Comment (asettle) [inline content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md]: Maybe put this in a note?

File: content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md [hunk 3]
```diff
@@ -11,152 +11,134 @@ product: Cloud Files
...
 ### Step 1
 
-Download the W3 Total Cache WordPress plugin from the 
-[W3 Total Cache website](http://wordpress.org/extend/plugins/w3-total-cache/). If you are 
-using any other caching plugins, deactivate and uninstall them. Ensure that WordPress has 
-write permissions to the **wp-content** directory.  You can do this from the server's 
+Download the W3 Total Cache WordPress plugin from the
+[W3 Total Cache website](http://wordpress.org/extend/plugins/w3-total-cache/). If you are
+using any other caching plugins, deactivate and uninstall them. Ensure that WordPress has
+write permissions to the **wp-content** directory.  You can do this from the server's
```
Comment (asettle) [inline content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md]: Remove double space before 'You can'

File: content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md [hunk 4]
```diff
@@ -11,152 +11,134 @@ product: Cloud Files
...
 
-If you already have a content delivery network (CDN) provider, go to the **Content Delivery 
-Network** tab and populate the fields and set your preferences. If you are using Rackspace 
-Cloud Files as your CDN, select Rackspace Cloud Files from the **Content Delivery Network** 
-tab.
-
-If you do not use the Media Library, you must import your images into the default locations. 
-Use the Media Library Import Tool on the **Content Delivery Network** tab to perform this 
-task.
+If you already have a content delivery network (CDN) provider, go to the **Content Delivery
```
Comment (asettle) [inline content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md]: probably don't need to have the full title and acronym again. Just acronym.

File: content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md [hunk 5]
```diff
@@ -11,152 +11,134 @@ product: Cloud Files
...
+Network** tab and populate the fields and set your preferences. If you are using Rackspace
```
Comment (asettle) [inline content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md]: using the Rackspace CDN

File: content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md [hunk 6]
```diff
@@ -11,152 +11,134 @@ product: Cloud Files
...
+CDN, log into your Rackspace account, and set up a CDN using the following procedure.
 
-If you do not have a CDN provider, you can still improve your website's performance using 
-the "self-hosted" method. On your own server, create a subdomain and matching DNS zone 
-record (for example, static.domain.com) and configure FTP options on the **Content Delivery 
-Network** tab accordingly. Use FTP to upload the appropriate files, using the available 
+If you do not have a CDN provider, you can still improve your website's performance using
+the "self-hosted" method. On your own server, create a subdomain and matching DNS zone
```
Comment (asettle) [inline content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md]: using a self-hosted method* - remove scare quotes

File: content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md [hunk 7]
```diff
@@ -11,152 +11,134 @@ product: Cloud Files
...
-
-6.	Click **Show** next to **API Key**. Note your username and API key from the Cloud Sites 
-   Control Panel. You will need these, as well as your Cloud Files credentials, when 
-   configuring W3 Total Cache.
+6.	Under **Caching Rules**, click on **Add Rule**, and specify the TTL length, and pathway
+        to your Wordpress content.
 
 Back in the WordPress control panel, perform the following steps:
 
 1.	Click on the **Content Delivery Network** (CDN) link to modify the settings.
```
Comment (asettle) [inline content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md]: ditto with the CDN thing

File: content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md [hunk 8]
```diff
@@ -11,152 +11,134 @@ product: Cloud Files
...
 
-2.	In the **Configuration** section, type your username and API key. You can also create 
-   the container for your website by typing the name in the container field and clicking 
-   **Create Container**. The hostname is automatically populated.
-
-3.	Under **SSL Support**, select (**auto detect**).
+2.	Under **SSL Support**, select (**auto detect**).
```
Comment (asettle) [inline content/cloud-files/accelerating-wordpress-with-cloud-files-cdn-and-the-w3-total-cache-plugin.md]: Do you select auto-detect or is it auto-selected?

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-128b111b49/how-to/)

**Comment by stephamon** [comment]
Closing because this article is basically a duplicate to another article that has been updated.

**Pull request closed without merging**